resources/[gameplay]/gus/gus_s.lua

```diff
@@ -451,7 +451,9 @@ addEventHandler('onPlayerFinish', root,
 				end
 			end
 		else
-			outputChatBox("there's no enough players online to record a streak. ("..getPlayerCount().."/"..requiredPlayersToRecordAStreak..")", root, 0, 255, 0)
+			if (rank == 1) then
+				outputChatBox("there's no enough players online to record a streak. ("..getPlayerCount().."/"..requiredPlayersToRecordAStreak..")", root, 0, 255, 0)
```

Comment:
Can you change this to:
`"There aren't enough players online to record this streak. ("..getPlayerCount().."/"..requiredPlayersToRecordAStreak..")"`
Thanks